scanapi/utils.py

```diff
@@ -23,6 +23,12 @@ def validate_keys(keys, available_keys, scope):
             raise InvalidKeyError(key, scope, available_keys)
 
 
+def validate_required_keys(keys, required_keys, scope):
+    if not set(required_keys) <= set(keys):
+        missing_keys = list(set(required_keys) - set(keys))
```

Comment:
```suggestion
        missing_keys = set(required_keys) - set(keys)
```

What do you think of send a set instance to MissingMandatoryKeyError init instead of a list, avoiding this casting? 

[`", ".join(sorted(missing_keys))`](https://github.com/scanapi/scanapi/pull/147/files#diff-14c555e2816bc0775ca3b7f2418fceeeR35)  works fine for sets too!

Reply:
I didn't know that. I'll update. Thank you!